docs/backuprestore-with-oadp.md

```diff
@@ -629,7 +665,7 @@ metadata:
 spec:
   ...
   oadpContent:
-  - name: oadp-cm-8g2mm56c2f
+  - name: oadp-cm
```

Comment:
above in the same IBU spec to change `v1alpha1` -> `v1`... seems it scaped from previous refacts